t/Couprex.t

```diff
@@ -22,12 +22,6 @@ ddg_spice_test(
         caller => 'DDG::Spice::Couprex'
     ),
 
-    'coupon codes for walgreens' => test_spice(
```

Comment:
Because it was the same test as the previous one. Maybe this PR wasn't the best option to do it, but I kept it as a separate commit :)